.pre-commit-config.yaml

```diff
@@ -23,6 +23,7 @@ repos:
   - id: clang-format
     args:
     - --style=Google
+    exclude: '^.*\.ipynb$'
```

Comment:
Is there any reason why to not pre-commit notebooks?

Reply:
our notebooks are python code, not C++ code, so there is no reason for clang-format to try format them.
Also, try to open one from the current devel: they are broken.
This break is even detected by the `check-json¿` hook from pre-commit/pre-commit-hooks.